lib/get-file-paths.js

```diff
@@ -4,7 +4,7 @@ var path = require('path');
 
 /**
  * Returns an object of file path data
- * 
+ *
  * @param {object}  file  a vinyl file object
  */
 exports.getFilePaths = function (file) {
```

Comment:
thanks for the suggestion @e2tha-e ! This file was WAAAAAAY too complicated. can you and @ericduran please check it now. drastically improved it seems